src/components/Navigator/NavigatorCardInner.vue

```diff
@@ -0,0 +1,35 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="navigator-card-inner"><slot /></div>
+</template>
+
+<script>
+export default { name: 'NavigatorCardInner' };
+</script>
+
+<style scoped lang="scss">
+@import 'docc-render/styles/_core.scss';
+
+$filter-height: 71px;
+
+.navigator-card-inner {
+  position: sticky;
+  top: $nav-height;
+  height: calc(100vh - #{$nav-height} - #{$filter-height});
```

Comment:
We shouldn't rest the `filter-height` when the navigator component is loading. This is why the UI doesn't reach the bottom of the page on its loading state. @dobromir-hristov pointed it out before.

Reply:
353c1dfd30aafdb57b02f74bb710b17f95403f67